src/components/burger-constructor/burger-constructor.tsx

```diff
@@ -0,0 +1,78 @@
+// @ts-nocheck
+
+import constructorStyle from './burger-constructor.module.css';
+import {ConstructorElement, Button, CurrencyIcon} from '@ya.praktikum/react-developer-burger-ui-components';
+import FillingIngredient from '../filling-ingredient/filling-ingredient'
+import PropTypes from 'prop-types';
+
+function BurgerConstructor(props) {
```

Comment:
также можно использовать деструктуризацию пропсов прямо тут
```suggestion
function BurgerConstructor({ burger: { upside, fillings, downside } }) {
```
и хорошо бы прописать тип для пропсов